files/fr/web/api/htmlmediaelement/audiotracks/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: HTMLMediaElement.audioTracks
+slug: Web/API/HTMLMediaElement/audioTracks
+browser-compat: api.HTMLMediaElement.audioTracks
+translation_of: Web/API/HTMLMediaElement/audioTracks
+---
+{{APIRef("HTML DOM")}}
+
+La propriété **`audioTracks`** en lecture seule sur les objets [`HTMLMediaElement`](/fr/docs/Web/API/HTMLMediaElement) renvoie un objet [`AudioTrackList`](/fr/docs/Web/API/AudioTrackList) répertoriant tous les objets [`AudioTrack`](/fr/docs/Web/API/AudioTrack) représentant les pistes audio de l'élément multimédia.
+
+L'élément multimédia peut être soit un élément [`<audio>`](/fr/docs/Web/HTML/Element/audio) ou un élément [`<video>`](/fr/docs/Web/HTML/Element/video).
+
+La liste renvoyée est en _live_ ; c'est-à-dire que lorsque des pistes sont ajoutées et supprimées de l'élément multimédia, le contenu de la liste change dynamiquement. Une fois que vous avez une référence à la liste, vous pouvez surveiller les modifications pour détecter l'ajout de nouvelles pistes audio ou la suppression de pistes existantes. Vous pouvez consulter les [`Event handlers`](/fr/docs/Web/API/AudioTrackList#event__handlers) (ou gestionnaires d'événements en français) au niveau de [`AudioTrackList`](/fr/docs/Web/API/AudioTrackList) pour en savoir plus sur les modifications apportées à la liste des pistes d'un élément multimédia.
```

Comment:
```suggestion
La liste renvoyée est dynamique&nbsp;; c'est-à-dire que lorsque des pistes sont ajoutées et supprimées de l'élément multimédia, le contenu de la liste change. Une fois que vous avez une référence à la liste, vous pouvez surveiller les modifications pour détecter l'ajout de nouvelles pistes audio ou la suppression de pistes existantes. Vous pouvez consulter les [gestionnaires d'évènements](/fr/docs/Web/API/AudioTrackList#évènements) associés à [`AudioTrackList`](/fr/docs/Web/API/AudioTrackList) pour en savoir plus sur les modifications apportées à la liste des pistes d'un élément multimédia.
```

Reply:
Ho, j'avais pas noté que l'ancre aussi était traduite; 👍🏿